packages/transaction-controller/jest.config.js

```diff
@@ -17,10 +17,10 @@ module.exports = merge(baseConfig, {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 83.91,
-      functions: 92.68,
-      lines: 95.35,
-      statements: 95.46,
+      branches: 71.88,
```

Comment:
I'm concerned we're dropping considerably on all fronts, is there any easy tests we can add to keep it at least at 90%?

Reply:
It could be my mistake to push that low locally, this is fixed in https://github.com/MetaMask/core/pull/1625/commits/6b989e35aebc9fad99a48c7e36424c876f90ae30